src/components/CurrencyInputPanel/CurrencyInputPanel.container.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useEffect, useMemo } from 'react'
+import debounce from 'lodash.debounce'
+
+import { CurrencyInputPanelComponent } from './CurrencyInputPanel.component'
```

Comment:
```suggestion
import { CurrencyInputPanelComponent } from './CurrencyInputPanel'
```
Not a big fan of .extenstions. Can make these names obvious for other files like 

currencyInputPanelTypes.ts
currencySeachModalUtilts.ts instead of CurrencySearchModal.hooks

all other files that end with .ts starts as small letters and main component with Caps and .tsx extension

Its also a personal suggestion. If everyone is onboard on current naming, I am ok

Reply:
Most of components in the dApp still suffer from having code in single `index.ts` file which contains default export. In my opinion it lowers dev experience as it makes searching/debugging harder. Splitting code into smaller files is a must and I think we can agree on it. When it comes to naming convention, as you said, it's a matter of personal preference, I just followed what I'm used to. 

We can move this discussion into broader audience and talk about it more but in this PR we're not going to change it as it's huge enough and needs to go to develop asap